proto/rill/local/v1/api.proto

```diff
@@ -74,6 +74,8 @@ message DeployValidationResponse {
   bool rill_org_exists_as_github_user_name = 13; // only applicable when user does not have any orgs
   repeated string rill_user_orgs = 14;
   string local_project_name = 15;
+  bool is_deployed = 16;
+  string deployed_project_id = 17;
```

Comment:
Could there be a case where `deployed_project_id` is set and yet `is_deployed` is false? 
Just trying to understand why `is_deployed` is separately required.

Reply:
Hmmm. This was an attempt to keep it consistent with other fields. Not necessary since it is just a single value now.